servicex_app/migrations/versions/1.5.7a.py

```diff
@@ -0,0 +1,25 @@
+"""Switch to index on email not sub
+
+Revision ID: v1_5_7a
```

Comment:
Why isn't this 1.6?

Reply:
It could be 1.6. I guess I didn't have a good sense how we want to number these (e.g. if I called it 1.6 then we'd need a 1.6a for anything else we wanted to land for that?) I actually wonder if we should just give these numbers or something instead of trying to suggest they're tied to specific releases.